core/src/main/scala/org/http4s/Request.scala

```diff
@@ -0,0 +1,233 @@
+package org.http4s
+
+import java.io.File
+import java.net.{InetAddress, InetSocketAddress}
+
+import cats.data.NonEmptyList
+import cats.Monad
+import cats.implicits._
+
+import org.http4s.headers.{Authorization, Host, `X-Forwarded-For`}
+import org.http4s.server.ServerSoftware
+import Message.messSyntax._
```

Comment:
Our imports are usually alphabetical.  It's the simplest rule that works across tools.  (With some grumbling around `_root_.io.circe` that we haven't quite worked out.)

Reply:
Absolutely. Sketch builds will need to be cleaned up. This was a giant experiment that fought me every step of the way.